packages/cardano-services/src/Handle/TypeOrmHandleProvider.ts

```diff
@@ -55,4 +60,33 @@ export class TypeOrmHandleProvider extends TypeormProvider implements HandleProv
       return handles.map((handle) => mapEntity(entities.find((entity) => entity.handle === handle)));
     });
   }
+
+  async getPolicyIds(): Promise<Cardano.PolicyId[]> {
+    return this.withDataSource(async (dataSource) => {
+      const columnName = 'policy_id';
+      const fetchPolicyIdsFromDB = async () => {
+        const distinctPolicyIds = await dataSource
+          .getRepository(HandleEntity)
+          .createQueryBuilder('handle')
+          .select(`DISTINCT ${columnName}`, columnName)
+          .getRawMany();
+        const policyIds = distinctPolicyIds.map((row) => row[columnName]);
+
+        if (policyIds.length === 0) {
+          throw new Error('Value from the database is 0');
```

Comment:
Can't we directly return `[]` from here?

Reply:
I don't think so, that would cache the `[]`

Reply:
Ok, thank you